docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -443,8 +532,30 @@ the :ref:`dronecan_fc_tutorial` tutorial.
 
 Arming and Arming Bypass
 ================================
+
+Standard Arming
+*****************
 DroneCAN can use the same advanced arming procedure as all other throttle sources. The details of this arming procedure are covered in the :ref:`manual_advanced_arming` section.
 
+Arming with ArmingStatus
+******************************
+Vertiq modules can arm and disarm based off of DroneCAN's :ref:`ArmingStatus message <arming_status>`. This means that whenever your flight controller broadcasts an ArmingStatus 
+``STATUS_FULLY_ARMED`` message, your module will transition from disarmed to armed, or if already armed, will remain armed. These transitions are not subject to the constraints 
+set by your :ref:`arming throttle region <arming_throttle_regions>`, so any ``STATUS_FULLY_ARMED`` message will arm your module. When your flight controller broadcasts a ``STATUS_DISARMED`` message, your module 
+will similarly transition from armed to disarmed.
+
+Whether your module arms via the ArmingStatus message is configured through the ``arming_by_arming_status`` parameter in the :ref:`DroneCAN node client <uavcan_node>`.
```

Comment:
We don't have it in there, but we will soon and it would be good to show it, can you show what setting this in Control Center will look like with a picture here? Even if its not from a currently released version of control center, an edited local version to show what it will look like could be good.